src/Directives.jl

```diff
@@ -95,11 +99,20 @@ function parsebracket end
 parsebracket(text :: AbstractString) = parsebracket(getdirective(text)...)
 parsebracket{D}(:: Directive{D}, text) = error("Unknown directive: '$D'")
 
+function parsebracket{D}(TD :: Directive{D}, arg, text)
+    if strip(arg) == ""
+        parsebracket(TD, text)
+    else
+        error("parsebracket not defined for directive '$D' with positional argument.")
+    end
+end
+
 function parsebracket(:: directive"docs", text)
+    eval(Expr(:toplevel, Expr(:using, :Lexicon)))
```

Comment:
I'm not immediately sold on `@{...}` (but maybe I'll warm to it). IMO `{{}}` is a standard in Markdown/html for generated content BUT this makes parsing a whole lot easier/feasible! `{{` and `}}` come up a lot in julia.

>  you can add plotting to docstrings 

holy crap. This seems amazingly useful.

A few of thoughts/questions:

1. Is the idea that any markdown files in docs/src will be built to docs/build with the expansions?
2. we should lob some default [css](https://github.com/JuliaGraphs/LightGraphs.jl/blob/master/doc/LightGraphs.css) + [js (for latex)](https://github.com/JuliaGraphs/LightGraphs.jl/blob/master/doc/mathjaxhelper.js) in there.
3. I think generating a README is something that'll be really useful.

I will continue to play... :)